src/Symfony/Component/Serializer/composer.json

```diff
@@ -16,12 +16,12 @@
         }
     ],
     "require": {
-        "php": ">=5.5.9"
+        "php": ">=5.5.9",
+        "symfony/property-access": "^3.1"
```

Comment:
Ah my bad, this one should not have been committed

On Tuesday, 5 July 2016, Guilhem N <notifications@github.com> wrote:

> In src/Symfony/Component/Serializer/composer.json
> <https://github.com/symfony/symfony/pull/19277#discussion_r69640059>:
>
> > @@ -16,12 +16,12 @@
> >          }
> >      ],
> >      "require": {
> > -        "php": ">=5.5.9"
> > +        "php": ">=5.5.9",
> > +        "symfony/property-access": "^3.1"
>
> not sure this should be moved as the serializer can be used without it
> with some normalizers.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/symfony/symfony/pull/19277/files/d4cdb00b32778cda9668d77919d3b5f99d770767#r69640059>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AE76gfsYm_9z4HQTvJDF56K-54UJMehVks5qSsp1gaJpZM4JD152>
> .
>